pages/1.11/installing/production/deploying-dcos/node-cluster-health-check/index.md

```diff
@@ -2,31 +2,30 @@
 layout: layout.pug
```

Comment:
This page should be moved to the major Administering Clusters section.  It would be a good idea to add a link to the final step of installation, for example this list: http://docs-review.mesosphere.com/1.11/installing/production/deploying-dcos/installation/#next-steps-enterprise-and-open-source-users

Reply:
@masonse and I have agreed on slack to re-think and move the "Node and cluster health checks" page to the "Administering clusters" page in Phase 2.